src/ddsrt/CMakeLists.txt

```diff
@@ -266,13 +266,12 @@ elseif(UNIX)
   endif()
 endif()
 
-if(${CMAKE_C_COMPILER_ID} STREQUAL "SunPro")
-  target_link_libraries(ddsrt INTERFACE socket nsl)
+if(iceoryx_binding_c_FOUND)
```

Comment:
Added this again, otherwise building with `-DBUILD_TESTING=ON` will not find the iceoryx header.

Reply:
I believe @eboasson is merging master and moving some stuff around. Let's fix this once the merge is complete and Iceoryx parts have been moved up to ddsc.